addons/website/static/tests/tours/edit_link_popover.js

```diff
@@ -184,5 +184,22 @@ wTourUtils.registerWebsitePreviewTour('edit_link_popover', {
         extra_trigger: 'iframe html:has(.o_edit_menu_popover)',
         run: function () {}, // it's a check
     },
+    {
+        content: "Ensure that a click on the link popover link opens a new window",
+        trigger: 'iframe .o_edit_menu_popover a.o_we_url_link[target="_blank"]',
+        extra_trigger: 'iframe .o_edit_menu_popover a.o_we_full_url[target="_blank"]',
+        run: function (actions) {
+            this.$anchor[0].addEventListener('click', (ev) => {
+                const originalPreventDefault = ev.preventDefault.bind(ev);
+                ev.preventDefault = () => {
+                    throw new Error(
+                        "The link popover should not be default prevented as to open a new tab"
+                    );
```

Comment:
Was the console.error not ok? It's the conventional way in all tours I think?

Reply:
Sorry missed [your message here](https://github.com/odoo/odoo/pull/114470#discussion_r1132619281), that's fine :+1: